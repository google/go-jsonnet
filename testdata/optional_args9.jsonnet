(function(x) x)(42, x=42)
