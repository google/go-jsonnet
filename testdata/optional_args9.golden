RUNTIME ERROR: Argument x already provided
-------------------------------------------------
	testdata/optional_args9:1:1-26	$

(function(x) x)(42, x=42)

-------------------------------------------------
	During evaluation	


